packages/components/src/navigation/Page.tsx

```diff
@@ -26,6 +26,7 @@ export const Page = ({
         {onBack !== undefined && (
           <button
             className="btn btn-link btn-back"
+            data-testid="page-back-button"
```

Comment:
Any reason to not use similar naming as the class? E.g.
```suggestion
            data-testid="btn-page-back"
```